src/Text/Pandoc/Writers/RST.hs

```diff
@@ -353,9 +353,24 @@ blockListToRST' :: PandocMonad m
                 -> [Block]       -- ^ List of block elements
                 -> RST m Doc
 blockListToRST' topLevel blocks = do
+  let transformLigature b1 b2@(BlockQuote _)
+         | isListBlock b1 = [b1, commentSep, b2]
+         | otherwise      = [b1, b2]
+        where
+          isListBlock (BulletList _)     = True
+          isListBlock (OrderedList _ _)  = True
+          isListBlock (DefinitionList _) = True
+          isListBlock _                  = False
+          commentSep  = RawBlock "rst" ""
```

Comment:
I'm confused: shouldn't the comment be `".."` and not `""`?

Reply:
I thought so as well, but tests proved me wrong. Off the top of my mind i imagine that a `RawBlock` renders an empty comment by default. Maybe this is because directives are similar to comments so a custom directive should be started with the same characters that would start an empty comment